src/components/common/Header/ProductHeader/StyledProductHeader.js

```diff
@@ -0,0 +1,79 @@
+import styled from "styled-components";
+
+export const HeaderBlock = styled.div`
+  background-color: #30363e;
+  min-height: 7.5vh;
+  font-family: "Roboto", sans-serif;
+  padding: 10px;
+  border-radius: 25px 25px 0 0;
+  a {
+    color: ${({ theme }) => theme.colors.NEUTRAL_2};
+  }
+`;
+
+export const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+export const Row = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  justify-content: space-between;
+`;
+
+export const HeaderTitle = styled.div`
+  color: #f1f4f5;
+  font-size: 3rem;
+  font-weight: 700;
+  font-style: bold;
+  margin-top: 1rem;
+  margin-left: 1rem;
+  line-height: 18px;
+`;
+export const HeaderSubtitle = styled.div`
+  color: #d3d4d6;
+  font-size: 1.5rem;
+  font-weight: 400;
+  font-style: italic;
+  margin-top: 1rem;
+  margin-left: 1rem;
+  line-height: 40px;
+`;
+export const ProfileId = styled.div`
+  color: #c4c4c4;
+  flex-basis: auto;
```

Comment:
I have never seen this used before, can you please tell me what it does?
I have disabled it and didn't make a difference.

Reply:
This was some extra css I left in there when i was playing around with flex box.